content/guides/user-guides/multi-cloud/storage-service-plan/_index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Storage Service Plan
+description: "User guide for Storage Service Plan(SSP) Management."
+weight: 10
+disableToc: false
+tags: ["user guide", "storage service plan", "multi-cloud"]
+---
+
+## Introduction to Storage Service Plan(SSP) management
+SODA is aimed at addressing the storage integration challenges of both the Cloud Native environment and traditional IT environment. SODA multi-cloud allows service providers(admin) or users to manage distributed cloud and on-premise environments on a single platform.
+
+
+SSP management allows service providers to abstract actual cloud or on-premise environments(backends) from their users.
+In other words, SSP is a policy for the users. A service provider can decide and create different types of SSP policy for different tenants of users.
+
+## Getting Started!
+This is a guide that shows how to install, configure, and use SSP management feature in a simple SODA setup.
+By following this guide you will be able to install SODA multi-cloud with creating backends and buckets, uploading and archiving objects.
+
+## Installing SODA multi-cloud
+
+### Download SODA-installer code
+
+```cassandraql
+git clone https://github.com/sodafoundation/installer.git
+cd installer/ansible
+# Checkout the latest release, give master by default or specify the branch
+git checkout master
+```
+
+### How to enable multi-cloud installation with SSP feature
+
+Install docker container to run multi-cloud in container environment.
+
+Update the file :
+
+a. group_vars/gelato.yml to install multi-cloud.
+```cassandraql
+# repository branch
+gelato_repo_branch: master
```

Comment:
this will apply only if someone is installing from repo and not release, right? if yes, then you can mention that this need to be changed only if repo install ..Or do we really need to put this ? :)